docs/development/in-shed-build-debugging.md

```
@@ -0,0 +1,186 @@
+   To exercise only one variant (faster on a small upper layer):
+
+   ```bash
+   ssh -F /tmp/publish-test-ssh shed-publish-test \
+       'cd /home/shed/work && VARIANTS=base ./scripts/publish-images-local.sh'
+   ```
+
+## Interpreting results
+
+PASS line:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)